catkit/emulators/tests/test_web_power_switch.py

```diff
@@ -25,13 +25,13 @@ def test_all_off():
 @pytest.mark.usefixtures("dummy_config_ini", "dummy_testbed_state")
 def test_individual_on():
     switch = WebPowerSwitch(config_id="web_power_switch")
-    switch.turn_on("dm1_outlet")
+    switch.turn_on("fpm_led")
```

Comment:
Update (I think - if I can remember correctly) because I update the fake config.ini copy.